hack/dev-env/dev-config.yaml

```diff
@@ -0,0 +1,561 @@
+---
```

Comment:
There are now two `dev-config.yaml` files (but slightly different number of lines). Is that intended?

Reply:
Whoops. Deleted the extra one.